packages/authentication/src/keys.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/authentication
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+export namespace LbKeys {
```

Comment:
I would prefer either `Keys` or `BindingKeys`. No need to prefix with `Lb` as the module is already `@loopback/authentication`